ddpui/api/alert_api.py

```
@@ -0,0 +1,385 @@
+        page = 1
+    if page_size < 1 or page_size > 100:
+        page_size = 20
+
+    try:
+        logs, total = AlertService.get_log(alert_id, orguser.org, page=page, page_size=page_size)
+    except AlertNotFoundError:
+        raise HttpError(404, "Alert not found") from None
+
+    total_pages = (total + page_size - 1) // page_size if total else 0
+    return LogListResponse(
+        data=[_build_log_out(l) for l in logs],
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)